com.endlessm.DiscoveryFeed.json.in

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id": "com.endlessm.DiscoveryFeed",
+    "build-options": {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
+    "command": "com.endlessm.DiscoveryFeed",
+    "finish-args": [
+        "--socket=x11",
+        "--share=ipc",
+        "--system-talk-name=com.endlessm.Metrics",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--filesystem=/var/lib/flatpak:ro",
+        "--filesystem=/var/cache/gnome-software:ro",
```

Comment:
This is probably not required, is it?

Reply:
It is - that's the path we get from g-s for the thumbnails.